### DISCUSSION TITLE
Pricing issue + ideas for improvement

### DISCUSSION THREAD

**Discussion by UkooLabs** [SixLabors/ImageSharp#1279] [Q&A] [answered]
Several points I gleamed from my company purchasing Image Sharp licences...

1) The unlimited license ($1999) when clicked asks for $4999.
2) It would be good to add a lifetime licence option if possible
3) My company also noted the Licence does not contain any form of Service Level Agreement to quote them "This license does not provide a Service Level Agreement, nor does it imply that SixLabors is responsible for successful deployment, maintenance nor business function of The Software. This license grants the right to all future product updates."
4) also https://sixlabors.com/blog/ is quite out of date (2019) it sort of scared my company into thinking things not recently updated until I told them how active you are etc

**Comment by JimBobSquarePants** [comment, marked as answer]
1. Fixed. $4999 is the correct price, I've updated it. Not sure how that happened.
2. Not going to happen. It allows big companies to exploit small ones,
3. There is no SLA. 
4. Many companies do not blog often.

**Reply by JimBobSquarePants** [reply]
I'm going to be really candid here. Technical issues and bugs, yes. Let's chat about them here (the pricing issue included) but how the company chooses to operate or offer licenses is NOT up for discussion and I see doing so as an insult.

**Reply by UkooLabs** [reply]
@JimBobSquarePants was not intending to cause any form of insult was just passing feedback from my company.

**Reply by JimBobSquarePants** [reply]
FWIW I've added a 2 day response time to the agreement plus listed "Enjoy prioritized feature requests and bug fixes" as a feature.
